forms-flow-components/src/components/CustomComponents/SortModal.tsx

```diff
@@ -119,7 +119,9 @@ export const SortModal: React.FC<SortModalProps> = React.memo(
           <CustomButton
             variant="primary"
             size="md"
-            disabled={!selectedOption || !selectedOrder}
+            disabled={!selectedOption ||
+              !selectedOrder ||
```

Comment:
"Extract these conditions into a variable declared before the returning the component